clients/cli/.gitignore

```diff
@@ -6,5 +6,4 @@ target/
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-# Generated protobuf files
-**/src/proto/*.rs
```

Comment:
Normally, I think its great to ignore these. But, it seems like the current intent is to commit them, so that a user does not need to have protoc installed to build the project.